itsm_api/views.py

```diff
@@ -833,48 +833,54 @@ def _post(self, data):
         """
         Process the addition of an attachment to a ticket.
         """
+        # Attachment files have a unique name generated for them when saving to server-side
+        # storage that looks like so:
+        #
+        # <attachment_id>__<filename>
+        #
+        # For example:
+        # 1__errors.log
+        #
+        # The directory structure of saved attachment files looks like this:
+        #
+        # attachment_storage/
+        #   <ticket_id>/
+        #     <unique_filename>
+        #
+        # For example:
+        #
+        # attachment_storage/
+        #   1/
```

Comment:
I think this docstring needs to be updated, now that the directories are called `ticket__<id>` ?

Reply:
Oh yes, updated to reflect proper directory storage